src/main/java/edu/harvard/h2ms/H2MSRestAppInitializer.java

```diff
@@ -1,26 +1,26 @@
-package edu.harvard.h2ms;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Import;
-import org.springframework.data.rest.core.event.ValidatingRepositoryEventListener;
-import org.springframework.data.rest.webmvc.config.RepositoryRestMvcConfiguration;
-
-import edu.harvard.h2ms.validator.EventValidator;
-
-/**
- * H2MS Rest Application
- * @version 1.0
- */
-@SpringBootApplication
-public class H2MSRestAppInitializer {
-
-	private static final Logger log = LoggerFactory.getLogger(H2MSRestAppInitializer.class);
-
-	public static void main(String[] args) {
-		SpringApplication.run(H2MSRestAppInitializer.class, args);
-	}
-}
+package edu.harvard.h2ms;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.icegreen.greenmail.util.GreenMail;
+import com.icegreen.greenmail.util.ServerSetupTest;
+
+/**
+ * H2MS Rest Application
+ * @version 1.0
+ */
+@SpringBootApplication
+public class H2MSRestAppInitializer {
+
+	private static final Logger log = LoggerFactory.getLogger(H2MSRestAppInitializer.class);
+	@SuppressWarnings("unused")
+	
+	public static void main(String[] args) {
+		GreenMail greenMail = new GreenMail(ServerSetupTest.SMTP);
+        greenMail.start();
+		SpringApplication.run(H2MSRestAppInitializer.class, args);

```

Comment:
Not sure if you looked at this, but having this in main doesn't strike me as very Spring-like, could it be a component instead? http://blog.netgloo.com/2014/11/13/run-code-at-spring-boot-startup/

That'd let us restrict it to run or not with @Profile annotations, for example.